CHANGELOG-chart.md

```diff
@@ -0,0 +1 @@
+- Chart and table.
```

Comment:
These go into the change log: We want something that explains the new feature without assuming context.